Event/include/Event/HcalHit.h

```diff
@@ -138,6 +155,9 @@ namespace ldmx {
             /** The number of PE estimated for this hit. */
             float pe_{0};
 
+            /** The minimum number of PE estimated for this hit, different from pe_ when you have two ended readout */
+            float minpe_{-99};
```

Comment:
It's a debug technique because technically minPE can be 0 and the hit will still be read out (where as PE cannot be 0 and read out).  If we want to change anything here, I would initialized `pe_` to -99 to be safe